pyproject.toml

```diff
@@ -41,12 +41,13 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 test = [
+    "sqlmodel",
```

Comment:
```suggestion
    "sqlacodegen['sqlamodel']",
```

Reply:
It's probably better to just change the CI to install `.[test,sqlmodel]` instead of defining `sqlmodel` dependency twice, so I'm gonna make this change instead

Reply:
Then you'd have to keep making these changes in both `pyproject.toml` every time you add something. I've used this method in my other projects and been happy with it. Where is the duplication here?

Reply:
The duplication would be defining `sqlmodel` in both tests and as the separate sqlmodel group. As you already Following the fix, there's no need to maintain a definition in separate files as the CI installes `.[test,sqlmodel]` and a versioned `sqlmodel` dependency is defined in a single place

Reply:
Also, corret me if I'm wron, but using `sqlacodegen['sqlmodel']` would result in a remote dependency being resolved, while we're aiming to relate to the local one

Reply:
No, it would not. Installers know what's being installed (`sqlacodegen`), and doing this will simply activate other extras in the same package.

Reply:
Ok, fixed according to your spec.